.changesets/feat_geal_coprocessor_latency_metric.md

```diff
@@ -0,0 +1,17 @@
+### add a metric tracking coprocessor latency ([Issue #2924](https://github.com/apollographql/router/issues/2924))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor_request_time
```

Comment:
```suggestion
apollo.router.operations.coprocessor.duration
```
Rationale:
`duration` seems to be the standard suffix when measuring the total time taken to response to a request vs `time` which is an accumulation of time during a period.

Two examples from the [conventions](https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/) are `http.server.duration` vs `system.cpu.time`

Otel namespaces are separated with `.` rather than underscore, and the prom exporter will convert for us so we can stick to the otel conventions.

Lastly, not sure `request` is adding value, so dropped it. I could be convinced otherwise if we fell that other metrics belong under the coprocessor namepsace, e.g.

```
apollo.router.operations.coprocessor.request.duration
apollo.router.operations.coprocessor.response.??
```
If not then let's just have:
`apollo.router.operations.coprocessor.duration`

Reply:
fixed in 17904c0b41ee44557034fe7c8f77ad678af02589